torch/autograd/function.py

```diff
@@ -173,7 +173,7 @@ def forward(ctx: Any, *args: Any, **kwargs: Any) -> Any:
         It must accept a context ctx as the first argument, followed by any
         number of arguments (tensors or other types).
 
-        The context can be used to store tensors that can be then retrieved
+        The context can be used to store tensors/inputs that can be then retrieved
```

Comment:
Why can it be used to saved inputs?
The extending doc explicitly says that you should use save_for_backward for inputs: https://pytorch.org/docs/stable/notes/extending.html

Reply:
I think this sentence means that you can do `ctx.foo =` to save anything that is not input/output.
But this is most likely not clear indeed.

Reply:
I am still not sure that this update makes sense. "tensors/inputs" is not really accurate because you can save pretty much everything you want, including intermediary result and now non-tensor outputs as well.